mu/modes/circuitpython.py

```diff
@@ -159,9 +137,8 @@ def get_volume_name(disk_name):
             # after warning the user.
             wd = super().workspace_dir()
             if self.connected:
-                m = _('Could not find an attached Adafruit CircuitPython'
-                      ' device.')
-                info = _("Python files for Adafruit CircuitPython devices"
+                m = _('Could not find an attached Circuit Python device.')
+                info = _("Python files for Circuit Python devices"
```

Comment:
Should `Circuit Python` be `CircuitPython`?
I don't know if both forms are valid.